internal/querycoordv2/meta/coordinator_broker.go

```diff
@@ -124,6 +124,22 @@ func (broker *CoordinatorBroker) GetRecoveryInfo(ctx context.Context, collection
 		return nil, nil, err
 	}
 
+	// fallback binlog memory size to log size when it is zero
```

Comment:
Is it a redundant code?
all `datapb.FieldBinlog` should be came from `datacoord`?
Do this fallback on  `catalog` of `datacoord` is enough?

Reply:
Compatible logic to make sure correct values from being obtained during rolling upgrades.